Makefile

```diff
@@ -45,10 +45,10 @@ dockerUp:
 	docker-compose --env-file ./manifests/claudie/.env up
 
 dockerDown:
-	docker-compose --env-file ./manifests/claudie/.env down
```

Comment:
Pls double-check the diffs in the Makefile, they seem to bring inconsistencies

Reply:
`docker-compose` has be deprecated and the project has moved under `docker` tool. More details can be found [here](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose)

Reply:
Shouldn't `dockerUp:` be updated too ?